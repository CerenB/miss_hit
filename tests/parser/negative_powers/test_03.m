% (c) Copyright 2019 Florian Schanda

function test_03()
  x = 2 ^ -2 + 1;  % (2 ^ -2) + 1
end
