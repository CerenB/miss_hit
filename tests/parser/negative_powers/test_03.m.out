----------------------------------------------------------------------
--  Parse tree for test_03.m
Root: Function_File
  Functions: Function_Definition
    Signature: Function_Signature
      Name: Identifier <test_03>
    Body: Sequence_Of_Statements
      Statements: Simple_Assignment_Statement
        LHS: Identifier <x>
        RHS: Binary_Operation +
          LHS: Binary_Operation ^
            LHS: Number_Literal <2>
            RHS: Unary_Operation -
              Expression: Number_Literal <2>
          RHS: Number_Literal <1>
----------------------------------------------------------------------
In test_03.m, line 3
| function test_03()
|                 ^ info: this token is not linked to the ast
In test_03.m, line 3
| function test_03()
|                  ^ info: this token is not linked to the ast
In test_03.m, line 4
|   x = 2 ^ -2 + 1;  % (2 ^ -2) + 1
|                 ^ info: this token is not linked to the ast
In test_03.m, line 5
| end
| ^^^ info: this token is not linked to the ast
MISS_HIT Summary: 1 file(s) analysed, everything seemes fine
